KsApi/models/graphql/adapters/Backing+BackingFragmentTests.swift

```diff
@@ -0,0 +1,534 @@
+import Apollo
+@testable import KsApi
+import XCTest
+
+final class Backing_BackingFragmentTests: XCTestCase {
```

Comment:
For now we just achieve the same test coverage as we currently have in https://github.com/kickstarter/ios-oss/blob/main/KsApi/models/graphql/adapters/Backing+GraphBackingTests.swift